pkgs/tools/misc/hdf5/default.nix

```
@@ -28,27 +31,37 @@ stdenv.mkDerivation rec {
   };
 
   passthru = {
-    inherit mpiSupport;
-    inherit mpi;
+    inherit
+      cppSupport
+      fortranSupport
+      fortran
+      zlibSupport
+      zlib
+      szipSupport
+      szip
+      mpiSupport
+      mpi
+    ;
   };
 
   outputs = [ "out" "dev" ];
 
   nativeBuildInputs = [ removeReferencesTo ]
-    ++ optional (gfortran != null) gfortran;
+    ++ optional fortranSupport fortran;
 
   buildInputs = []
```

```suggestion
  buildInputs =
```
lib.optional(s) returns an empty [] when false.